Neo4j.Driver/Neo4j.Driver.Tests/ConnectionPoolTests.cs

```diff
@@ -84,7 +84,7 @@ public void ShouldBlockWhenMaxPoolSizeReached()
 
                 pool.NumberOfIdleConnections.Should().Be(0);
                 pool.NumberOfInUseConnections.Should().Be(2);
-                timer.ElapsedMilliseconds.Should().BeGreaterOrEqualTo(1000);
+                Math.Round(timer.Elapsed.TotalSeconds).Should().BeGreaterOrEqualTo(1);
```

Comment:
Is it correct to have Math round over it? Shouldn't it just greater than 1?
I am thinking if it is enough to only have `timmer.Elapsed.TotalSeconds.Should().BeGreaterOrEqualTo(1);`?

Reply:
The one you suggested was failing for some other tests - the problem lies on the fact that `timer.Elapsed.TotalSeconds` reports slightly less than `1` (i.e. `0.998...`) mostly on our brand new linux build agents. 

It is similar to the other elapsed time assertions we made on this class - I changed all of them to be rounded to the nearest integral. They all have the same pattern that sometimes they fail with slightly less elapsed times than expected, which I believe is a result of timer resolution on different platform, hardware and/or operating system.

Reply:
Would it be to inaccurate for this number? Maybe `Math.Round(timer.Elapsed.TotalSeconds*10).Should().BeGreaterOrEqualTo(10)`? Or put in a new method such as 
```
double round(double value, double accuracy) {
  return Math.Round(value/accuracy)*accuracy;
}
round(timer,Elapsed.TotalSeconds, 0.1).Should().BeGreaterOrEqualTo(1);
```

Reply:
I got rid of `Math.Round` calls and instead add an overloaded `BeGreaterOrEqualTo` assertion that takes an accuracy parameter.